03_letters/joshcarp/letters.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+)
+
+type StringSlice []string
```

Comment:
I think you might be doing this so that you can use the `sort.Sort` function, but I don't think creating a custom type which is just a wrapper around a normal type simplifies the solution. 

You should be able to use a normal `[]string` and use the `sort.Slice` function to achieve the same outcome. This will simplify your solution and you won't need this custom wrapper code.